tests/templates/test_broadcast.py

```diff
@@ -270,24 +257,27 @@ def circuit():
         with pytest.raises(ValueError, match="'parameters' must be either of type None or "):
             circuit()
 
-    @pytest.mark.parametrize("function, wires, target", [(wires_pyramid, [8, 2, 0, 4, 6, 2],
-                                                          [[8, 2], [0, 4], [6, 2], [2, 0], [4, 6], [0, 4]]),
+    @pytest.mark.parametrize("function, wires, target", [(wires_pyramid, [8, 2, 0, 4, 6, 1],
```

Comment:
any significance to the 2 -> 1 change?

Reply:
Yes, the test was sloppy and used a wires sequence that was not allowed because the 2 got repeated. That is now a problem, when ``broadcast`` turns the sequence into Wires and does proper checks.

Reply:
Nice checks in places for `Wires`! :)